packages/application-components/src/components/modal-page/README.md

```diff
@@ -14,41 +14,42 @@ Modal pages are controlled components used to render a page using a modal contai
 
 ```js
 <ModalPage
+  title="Lorem ipsus"
   isOpen={true}
   onClose={handleClose}
-  title="Lorem ipsus"
   subtitle={<Text.Body>{'Lorem ipsus ...'}</Text.Body>}
-  components={{
-    actions: (
-      <Spacings.Inline alignItems="flex-start">
-        <SecondaryButton label="Cancel" onClick={handleReset} />
-        <PrimaryButton
-          label="Save"
-          isDisabled={isDisabled}
-          onClick={handleSubmit}
-        />
-      </Spacings.Inline>
-    ),
-  }}
+  headerActions={
+    <Spacings.Inline alignItems="flex-start">
```

Comment:
Didnt we agree that splitting actions into two would be good?

Reply:
Yes, however after further analysing the designs and use cases for this, I found cases where there are 3 different buttons, and where the type of the buttons (primary/secondary, iconButton) also differs.
So, given this need for flexibility which would require a bunch of different parameters, I decided to keep the actions has a render prop.

Reply:
> actions has a render prop.

It's not a render-prop. That would imply it being an fn receiving arguments. Is that what we need here?

Reply:
Sorry, you're right, not a render prop. It's a node which its rendered aligned in the right side of the Header section of the Modal.